app/routes.py

```diff
@@ -1 +1,274 @@
-from flask import Blueprint
\ No newline at end of file
+from os import abort
+from app import db
+from app.models.task import Task
+from flask import Blueprint, jsonify, abort, make_response, request
+from sqlalchemy import asc, desc
+from datetime import datetime
+# from slack_sdk import WebClient
+# from slack_sdk.errors import SlackApiError
+import os
+import requests
+from app.models.goal import Goal
+import json
+
+
+
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+def validate_model(cls, model_id):
+    try:
+        model_id = int(model_id)
+    except:
+        abort(make_response({"message":f"task {model_id} invalid"}, 400))
+
+    model = cls.query.get(model_id)
+    
+    if not model:
+        abort(make_response({"message":f"{cls.__name__} {model_id} not found"}, 404))
+    
+    return model
+
+
+@tasks_bp.route("", methods=["POST"])
+def create_task():
+    request_body = request.get_json()
+    try:
+        new_task = Task.from_dict(request_body)
+
+    except:
```

Comment:
Love how you implemented a `try/except` block, here! Like we validated models, could we also make a general function that validates request bodies? Here's an example:
```python
def validate_request_body(request_body, keys):
    for key in keys:
        if not request_body.get(key): 
            abort(make_response({
                'Invalid Data': f'missing key: {key}'
            }, 400))

    return True
```
We can pass in the `request_body` and a list of strings that are `keys` and then check to see if those keys are present.